migrations/20220321220618-rename-virtual-card-epire-date-to-expiry-date.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+import { omit } from 'lodash';
+
+import { crypto } from '../server/lib/encryption';
+
+async function renameFieldInVirtualCardsPrivateData(queryInterface, from, to) {
+  const [virtualCards] = await queryInterface.sequelize.query(
+    `SELECT "id", "privateData" FROM "VirtualCards" WHERE "privateData" IS NOT NULL;`,
+  );
+  console.info(`Renaming '${from}' to '${to}' for ${virtualCards.length} VirtualCards...`);
+  try {
+    await queryInterface.sequelize.transaction(async transaction => {
+      for (const virtualCard of virtualCards) {
+        const originalPrivateData = JSON.parse(crypto.decrypt(virtualCard.privateData));
+        if (!(from in originalPrivateData)) {
+          console.debug(`'${from}' not in 'privateData' for VirtualCard with id=${virtualCard.id}`);
+          continue;
+        }
+        const privateData = {
+          ...omit(originalPrivateData, [from]),
+        };
+        privateData[to] = originalPrivateData[from];
+        const encryptedPrivateData = JSON.stringify(crypto.encrypt(privateData));
```

Comment:
This fails with `RangeError: Invalid array length`.`crypto.encrypt` expects a string

```es6
const encryptedPrivateData = crypto.encrypt(JSON.stringify(privateData));
```